refarch-frontend/src/main.ts

```diff
@@ -0,0 +1,10 @@
+import { createApp } from "vue";
+
+import { registerPlugins } from "@/plugins";
+import App from "./App.vue";
```

Comment:
We should always import files using the `@/..` shorthand provided by Vite. This should also be possible in non Vue SFCs.

Reply:
#79